Code/Framework/AzFramework/AzFramework/Physics/Material/PhysicsMaterialSystemComponent.cpp

```diff
@@ -61,6 +61,14 @@ namespace Physics
 
     void MaterialSystemComponent::Deactivate()
     {
+        for (auto& assetHandler : m_assetHandlers)
+        {
+            if (auto materialAssetHandler = azrtti_cast<AzFramework::GenericAssetHandler<MaterialAsset>*>(assetHandler.get());
+                materialAssetHandler != nullptr)
+            {
+                materialAssetHandler->Unregister();
+            }
+        }
```

Comment:
Good catch! This might be related to this issue https://github.com/o3de/o3de/issues/10825 

This means there is also a `Unregister` missing for [this one too](https://github.com/o3de/o3de/blob/76074304573950182e2d48283ff4b01b798bab03/Gems/PhysX/Code/Editor/Source/Components/EditorSystemComponent.cpp#L81), can you add it as well please? Thanks!

Reply:
Thanks!

Reply:
Done: https://github.com/o3de/o3de/pull/14824/commits/d5962dd52fddb8e36c668e715dc26610799f19e1